Tools/CISupport/ews-build/events.py

```diff
@@ -313,14 +313,19 @@ def stopService(self):
 
 class GitHubEventHandlerNoEdits(GitHubEventHandler):
     ACTIONS_TO_TRIGGER_EWS = ('opened', 'synchronize')
+    OPEN_STATES = ('open',)
 
     def _get_commit_msg(self, repo, sha):
         return ''
 
     def handle_pull_request(self, payload, event):
         pr_number = payload['number']
         action = payload.get('action')
+        state = payload.get('state')
         if action not in self.ACTIONS_TO_TRIGGER_EWS:
             log.msg('Action {} on PR #{} does not indicate code has been changed'.format(action, pr_number))
             return ([], 'git')
+        if state not in self.OPEN_STATES:
+            log.msg("on PR #{} is '{}', triggering nothing".format(action, state))
```

Comment:
This reads weird.
Also, does this goes to twistd.log?

Reply:
It does go to twistd.log. And changed the log message, it will now be:
`PR #{} is 'closed', which triggers nothing`

If a new state is added, it would be that state instead of "closed"